cmd/storage-rest-client.go

```diff
@@ -268,7 +276,13 @@ func (client *storageRESTClient) ReadAll(volume, path string) ([]byte, error) {
 		return nil, err
 	}
 	defer http.DrainBody(respBody)
```

Comment:
Need to include the bytes transferred in the DrainBody function too.

Reply:
It doesn't make sense to do this, since we already `ioutil.ReadAll`. This func call is acting as a closer essentially.

Reply:
True, but there are other usages of this function, where it may not be doing a ReadAll - we needed the DrainBody to be able to correctly reuse the connection object - see the comments at the func def. Since we are reading data from the connection, it makes sense to include it in the counts.

Reply:
I know why DrainBody is being called, but I don't you understand what's going on here. 

In each and every one of the functions, where network stats are counted, there is no need to count the bytes the bytes that are being drained.

Reply:
Ah, ok I see what you mean about the counting already taking the drain body into consideration.